src/main/webapp/repo-bootstrap/global/blueprints/1000_website_editorial/config/studio/content-types/page/article/form-definition.xml

```diff
@@ -21,6 +21,8 @@
     <objectType>page</objectType>
     <content-type>/page/article</content-type>
     <imageThumbnail>page-article.png</imageThumbnail>
+    <quickCreate>false</quickCreate>
```

Comment:
Articles in Editorial BP should be quick creatable and should automatically go in the right folders.

Reply:
Done